creme/metrics/_confusion_matrix.pyx

```diff
@@ -0,0 +1,232 @@
+import warnings
+
+import numpy as np
+cimport numpy as np
+
+from collections import defaultdict
+
+cdef bint boolean_variable = True
```

Comment:
I can't this being used anywhere :)

Reply:
It is used here :)

https://github.com/creme-ml/creme/blob/a61a9597c360687d911e4a90eaded8f8088b06cd/creme/metrics/_confusion_matrix.pyx#L168

Reply:
Lol sorry I don't understand: `boolean_variable` is the name of the boolean variable that is declared, but I don't see any reference to `boolean_variable` in the rest of the file.

Reply:
My bad, you are right, that was completely unrelated and unused.